app/src/main/res/values/preference_keys.xml

```diff
@@ -12,7 +12,7 @@
     <string name="preference_key_app_channel">channel</string>
     <string name="preference_key_language_mru">languageMru</string>
     <string name="preference_key_language_app">languageApp</string>
-    <string name="preference_key_multilingual_search_tutorial_enabled">multilingualSearchTutorialEnabled</string>
+    <string name="preference_key_multilingual_search_tooltip_shown">multilingualSearchTooltipShown</string>
```

Comment:
If we change the actual value of the key, it means that the user will see the tooltip again, even if they've already seen it. Are we sure we want to do that?

Reply:
You are correct. Fixed.